channeldb/payments_test.go

```diff
@@ -422,3 +423,63 @@ func TestQueryPayments(t *testing.T) {
 		})
 	}
 }
+
+// appendDuplicatePayment adds a duplicate payment to an existing payment. Note
+// that this function requires a unique sequence number.
```

Comment:
The only reason we need this code is to reproduce a legacy database right? Maybe explain that in a comment